Bixal/move.mil
# Feature/156819258/complete accordion theming

Fair warning @rich-allen-gov: this is a bigger pull request than normal. I started with a relatively simple set of config changes to add the three new accordion sections to the [Moving Tips page](https://www.move.mil/moving-guide/tips), and realized once placed that a different approach to the accordion views was necessary to get the images integrated properly. The resulting changes, listed below, should completely take care of the accordions on the FAQ and Moving Tips pages and their associated stories.  

## Checklist

I have…

      against my changes

## Summary of Changes

This pull request…

- Adds three new displays to the FAQ view, each associated with a section of the Moving Tips page
- Changes the view structure of these blocks and the five FAQ blocks from Fields to a Content teaser display.
- Creates three new blocks corresponding to those new views and places them on the page layout, just below the main content block and restricted to the Moving Tips path.
- Adds a new twig file for the FAQ teaser display tied to the views, matching the accordion structure and including the <button> element as previously discussed.
- Adds new blue +/- icon files to match those used on the live site's accordions.
- Adds sass to align images and text within the accordion header buttons and reference the new blue +/- icons.
- Adds a quick fixes file and includes a short-term hack to make local error messages from missing dependencies way less annoying (we'll get rid of this before deployment to production, and I'll ask Jared to create a ticket for that shortly).

## Testing

To verify the changes proposed in this pull request…

1. Pull code
1. Import config
1. run `npm run build`
1. Check the [FAQ](http://move.mil.localhost:8000/moving-guide/tips) and [Moving Tips](http://move.mil.localhost:8000/faqs) pages on your local to see accordions in action (latest db from alpha might be necessary for full integrity)

## Screenshots

### Moving Tips page (local as implemented):
<img width="964" alt="screen shot 2018-04-16 at 5 26 59 pm" src="https://user-images.githubusercontent.com/29130580/38836257-78560902-419b-11e8-8ae7-a5848695c538.png">

### FAQ page (local as implemented):
<img width="1068" alt="screen shot 2018-04-16 at 5 30 06 pm" src="https://user-images.githubusercontent.com/29130580/38836365-da93a0e8-419b-11e8-997f-7d91146468b6.png">
